src/types.ts

```diff
@@ -158,33 +158,33 @@ export interface ValidatorEntryBuffer extends Omit<ValidatorEntry, 'address'> {
 	address: Buffer;
 }
 
-export interface SentVote {
-	delegateAddress: string;
+export interface Stake {
+	validatorAddress: string;
 	amount: bigint;
-	voteSharingCoefficients: SharingCoefficients[];
+	stakeSharingCoefficients: SharingCoefficients[];
 }
 
-export interface Voter {
+export interface Staker {
 	address: string;
-	sentVotes: {
-		delegateAddress: string;
+	sentStakes: {
+		validatorAddress: string;
 		amount: bigint;
 	}[];
 	pendingUnlocks: {
-		delegateAddress: string;
+		validatorAddress: string;
 		amount: bigint;
-		unvoteHeight: number;
+		unstakeHeight: number;
 	}[];
 }
 
 export interface GenesisDataEntry {
 	initRounds: number;
-	initDelegates: string[];
+	initValidators: string[];
 }
 
 export interface DPoSStoreEntry {
```

Comment:
Should be PoSStoreEntry.
Moreover, LIP also contains [Eligible Validators Substore](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#eligible-validators-substore-1) and [Previous Timestamp Substore](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#previous-timestamp-substore-1), not sure whether this is already in SDK and whether it should be mentioned here.

Reply:
Updated! Also, regarding the schemas you mentioned here are part of SDK implementation.